nautilus_core/network/src/http.rs

```diff
@@ -78,6 +79,7 @@ impl InnerHttpClient {
         url: String,
         headers: HashMap<String, String>,
         body: Option<Vec<u8>>,
+        timeout_sec: Option<u64>,
```

Comment:
nit: I'd probably go with `timeout_secs`, but don't worry about updating that and running CI again - I'll update on `develop`.

Reply:
Okay perfect